dashboards/ansible-tower/ansible-tower.json

```diff
@@ -0,0 +1,801 @@
+            "visualization": {
+              "id": "viz.table"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT uniques(tower_version), uniques(ansible_version) SINCE 3 months ago"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Services Running on Host",
+            "layout": {
+              "column": 1,
+              "row": 3,
+              "width": 4,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.pie"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT count(*) FACET job"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "AWX Hosts",
+            "layout": {
+              "column": 5,
+              "row": 3,
+              "width": 2,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT sum(awx_hosts_total) AS '' SINCE 1 month ago"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Successful Jobs",
+            "layout": {
+              "column": 7,
+              "row": 3,
+              "width": 2,
+              "height": 2
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT count(*) WHERE entity.guid = 'MzUxMDYxM3xJTkZSQXxOQXw4MjYzOTA3MzM5MjAwNjEyMDE3' OR (status = 'successful' AND prometheus_server = 'prometheus_remote' AND job = 'tower') SINCE 1 HOUR AGO"
```

Comment:
It looks like the `entity.guid` is hard-coded in this query. We should remove this to ensure that it works the same for all users that install it.

Reply:
done.. removed it 👍